src/pages/home/report/ReportActionsView.tsx

```diff
@@ -237,24 +237,6 @@ function ReportActionsView({
     const oldestReportAction = useMemo(() => reportActions?.at(-1), [reportActions]);
     const hasCreatedAction = oldestReportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED;
 
-    useEffect(() => {
-        if (reportActionID) {
-            return;
-        }
-
-        const interactionTask = InteractionManager.runAfterInteractions(() => {
```

Comment:
What is this interactionTask for and is it ok that we won't have it in ReportScreen?

Reply:
I found that it's added from this [performance improvement PR](https://github.com/Expensify/App/pull/38255). Should we put it inside the interaction task?
https://github.com/Expensify/App/blob/1c651a1ac7c6af25df9459861ba4128e8d50e2fb/src/pages/home/ReportScreen.tsx#L468-L471

Reply:
Please do a test with it in the interaction task and without, and show the results with videos and or timing logs.

Reply:
Updated to put it inside interaction task.

Without interaction task:

https://github.com/Expensify/App/assets/50919443/37915ee0-b09a-4f1d-a7e5-38c6bde741f3

With interaction task:

https://github.com/Expensify/App/assets/50919443/fc492e63-7bdc-43fc-a305-d087763f4396


Here is the log result of `switch_report` timing.
https://github.com/Expensify/App/blob/ee8f9c661f269f405a5ae8ba47ea07f42a306c6c/src/pages/home/report/ReportActionsView.tsx#L385-L401


| Without interaction task  | With interaction task |
| ------------- | ------------- |
| 1033  | 1025  |
| 879  | 994 |
| 1152 | 1078 |
| 1692 | 1652 |
| Avg: 1189 | Avg: 1187.25 |

Reply:
Interesting thanks so much! The timing results don't look too different, but the video shows it's better with the interaction task.